pkg/porter/run.go

```diff
@@ -102,6 +105,31 @@ func (p *Porter) Run(opts RunOptions) error {
 		return err
 	}
 
+	//Update the runtimeManifest images with the bundle.json and relocation mapping (if it's there)
```

Comment:
nit: In a follow up, it would be nice to wrap this all up in a function in this same file so that `Run` continues to be pretty readable at a high level. Maybe `handleRelocationImages` or something.

Reply:
good point, i'll do a follow up. I *think* i want to do a refactor of a bunch of the logic here

Reply:
Yeah based on the touched file list you showed me, a refactor doesn't seem like a bad idea if you have ideas on how to improve the abstraction.